src/app/views/criteria/create/create.component.ts

```diff
@@ -84,16 +108,45 @@ export class CreateComponent {
         indicatorID: this.indicatorID,
       })
       .subscribe({
-        next: () => {
-          this.router.navigate(['/criteria']);
+        next: (response) => {
+          this.toggleToast('El criterio se ha creado exitosamente', true); // Mostrar toast de éxito
+          setTimeout(() => {
+            this.router.navigate([`criteria`]); 
+          },1500)
         },
         error: (error) => {
+          this.toggleToast('Error al crear criterio', false); 
```

Comment:
```suggestion
          this.toggleToast(error.message, false); 
```
o....
```suggestion
          this.toggleToast(error.error.message, false); 
```
Como sea que lo envie el backend